addons/zylann.scatter/tools/palette.gd

```diff
@@ -1,7 +1,7 @@
 tool
 extends Control
 
-signal pattern_selected(pattern_index)
+signal patterns_selected(patterns)
```

Comment:
You changed `patterns` to be a list of paths, so better rename the argument `pattern_paths`

Reply:
Done.